src/controllers/userController.js

```diff
@@ -0,0 +1,48 @@
+const User = require('../models/User');
+const jwt = require('jsonwebtoken');
+
+// create token (we use it here after updating user's data )
+const createJWT = ({ payload }) => {
+  const token = jwt.sign(payload, process.env.JWT_SECRET, {
+    expiresIn: process.env.JWT_LIFETIME,
+  });
+  return token;
+};
+
+const getSingleUser = async (req, res) => {
+  const user = await User.findOne({ _id: req.params.id }).select('-password');
```

Comment:
are there any other fields that we don't need to return the frontend? I'm thinking maybe `role` and `creditCardInfo` don't need to be returned in most cases? If you end up implementing a `formatUserForFrontend` function ([PR 1](https://github.com/Code-the-Dream-School/dd-prac-team4-back/commit/b48b2d231e4183b4ca3b283bc055a0eafb657021#r123402026) / [PR2](https://github.com/Code-the-Dream-School/dd-prac-team4-back/pull/6#discussion_r1282047069)), maybe you can just reuse that here?

Reply:
@akosasante done